base/install_motor.sh

```diff
@@ -34,3 +36,33 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+#Add pmac module
+
+cd ..
+
+download_github_module dls-controls pmac $PMAC_VERSION
+
+rm -f pmac/configure/RELEASE.local.linux-x86_64
+rm -f pmac/configure/RELEASE.linux-x86_64.Common
+rm -f pmac/configure/CONFIG_SITE.linux-x86_64.Common
+
+echo "
+BUILD_IOCS=NO
+USE_GRAPHICSMAGICK=NO
+
+SSH             = $(pkg-config --variable=prefix libssh2)
+SSH_LIB         = $(pkg-config --variable=libdir libssh2)
+SSH_INCLUDE     = -I$(pkg-config --variable=includedir libssh2)
+
+WITH_BOOST = NO
+
+" >> pmac/configure/CONFIG_SITE
```

Comment:
Nit: don't need this empty line in the middle

(if it was because of the newline comment, `echo` will add it by default)